manifests/0000_25_kube-scheduler-operator_01_config.crd.yaml

```diff
@@ -77,9 +76,16 @@ spec:
               description: 'unsupportedConfigOverrides holds a sparse config that
                 will override any previously set options.  It only needs to be the
                 fields to override it will end up overlaying in the following order:
-                1. hardcoded defaults 2. observedConfig 3. unsupportedConfigOverrides'
-              nullable: true
+                1. hardcoded defaults 2. observedConfig 3. unsupportedConfigOverrides
+                +nullable'
```

Comment:
this is wrong.

Reply:
@sttts these +comments should be filtered out by the generator. Looks like crd-schema-gen is using the `master` branch of our `controller-tools` fork. That's probably an issue, since I believe we were working on the `origin-4.1-kubernetes-1.13.4` branch there right?

Reply:
@damemi & @sttts - Can you elaborate on what's wrong. I can try fixing them in generator..